app/scripts/platforms/extension.js

```diff
@@ -0,0 +1,23 @@
+const extension = require('extensionizer')
```

Comment:
`extensionizer` returns a singleton, it will be global throughout the app.  You're then wrapping it in a class, so each module that uses it gets a copy of that wrapper class.

Fortunately, each instance should still have a reference to the same singleton, so requiring this class and creating an instance should be an appropriate way of globally accessing platform features.